packages/code-du-travail-frontend/src/ReferencesJuridiques/components/HierarchieBloc.js

```diff
@@ -22,7 +22,11 @@ class HierarchieBloc extends React.PureComponent {
 
     const items = [
       {
-        title: "Voir les articles du code du travail concernés",
+        title: (
+          <StyledSpan>
```

Comment:
Peut être qu'il faudra intégrer ce composant à l'UI si on s'en sert à plusieurs endroits

Reply:
C'est noté ! Mais il se pourrait que ces deux là divergent dans un avenir proche aussi !